README.md

```diff
@@ -133,20 +134,21 @@ You may add Gatsby Image support to any GraphQL Type describing a Cloudinary ass
 }
 ```
 
-To find the GraphQL Type describing your Cloudinary assets use the buildt in [GraphiQL exlorer](https://www.gatsbyjs.com/docs/how-to/querying-data/running-queries-with-graphiql/). Either by hovering over the field describing the asset, or by looking it up in the "Documentation Explorer".
+To find the GraphQL Type describing your Cloudinary assets use the built in [GraphiQL exlorer](https://www.gatsbyjs.com/docs/how-to/querying-data/running-queries-with-graphiql/). Either you hover over the field describing the asset, or you look in the "Documentation Explorer".
```

Comment:
What about "Either hover over the field describing the asset, or look in the "Documentation Explorer".?

Reply:
👍